exporter/sapmexporter/config.go

```diff
@@ -40,3 +49,43 @@ type Config struct {
 	// Note that right now this is only used when the server responds with HTTP 429 (tries to rate limit the client)
 	MaxRetries *uint `mapstructure:"max_retries"`
 }
+
+func (c *Config) validate() error {
+	if c.Endpoint == "" {
```

Comment:
This is a general issue for the collector: we are not being consistent with usage of `url` versus `endpoint` for the collector. @ccaraman just pointed this somewhere else. Not to change it on this PR but it will be nice to settle on a pattern that applies to all exporters at least.

Reply:
Here is the GH issue tracking converting to 'endpoint' away from 'url' https://github.com/open-telemetry/opentelemetry-collector/issues/472